documentation/docs/getting_started/installation/java/local_setup.md

```diff
@@ -0,0 +1,92 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: This section will guide in setting up the Integration Service API using Java.
+keywords:
+- Java
+- setup API
+- configure
+- .env
+---
+
+# Local Setup
+
+:::caution
+The [Integration Services Java library](https://github.com/albydeca/iota-is-sdk) is still in BETA. Please note that not everything may yet run smoothly.
+:::
+
+This section will guide in setting up the Integration Services API using Java.
+
+## Requirements
+
+See project's [POM](https://github.com/albydeca/iota-is-sdk/blob/main/pom.xml). Moreover, the following will be required:
+
+* JDK 16 (recommended)
+* A reference to an instance of the [Integration Services API](https://github.com/iotaledger/integration-services)
+* Maven
+
+## Download and Build the Project
+
+Build from source
+```bash
+git clone git@github.com:albydeca/iota-is-sdk.git
+cd iota-is-sdk
+mvn clean install
+```
+
+or download JAR from [MVNRepository](https://mvnrepository.com/artifact/net.gradbase/iota.is.sdk/0.0.1)
+
+or simply add to your POM:
+```xml
+<!-- https://mvnrepository.com/artifact/net.gradbase/iota.is.sdk -->
+<dependency>
+    <groupId>net.gradbase</groupId>
+    <artifactId>iota.is.sdk</artifactId>
+    <version>0.0.1</version>
+</dependency>
+```
+
+## Configuration
+
+Please set up the following files in order to generate the JAR and run the code locally:
+
+- `env.properties` - with the following structure:
+  ```
+  api-key=XXXXXXX
+  api-version=vX.X
+  api-url=XXXXXXX
+  identity-file=adminIdentity.json
+- `adminIdentity.json` - will contain the admin identity object (json file with elements `doc` and `key`)
+
+Then, perform the following steps:
+```bash
+mvn clean install
+```
```

Comment:
I think you are missing triple ticks here somewhere.

Reply:
indeed :)